ui/public/locales/pt_BR.json

```diff
@@ -1404,146 +1836,191 @@
+"label.remove.vmware.datacenter": "Remover datacenter VMware",
+"label.remove.vpc": "Remover VPC",
 "label.remove.vpc.offering": "Remover oferta VPC",
 "label.removing": "Removendo",
-"label.removing.user": "Removendo Usu\u00e1rio",
+"label.removing.user": "Removendo usu\u00e1rio",
 "label.replace.acl": "Substituir ACL",
-"label.replace.acl.list": "Substituir Lista ACL",
+"label.replace.acl.list": "Substituir lista ACL",
+"label.report.bug": "Reportar um problema",
 "label.required": "Obrigat\u00f3rio",
 "label.requireshvm": "HVM",
-"label.requiresupgrade": "Requer Atualiza\u00e7\u00e3o",
+"label.requiresupgrade": "Requer atualiza\u00e7\u00e3o",
 "label.reserved.system.gateway": "Gateway de sistema reservado",
-"label.reserved.system.ip": "IP de Sistema Reservado",
+"label.reserved.system.ip": "IP de sistema reservado",
 "label.reserved.system.netmask": "M\u00e1scara de rede reservada do sistema",
-"label.reservediprange": "Faixa de IP Reservada",
+"label.reservediprange": "Faixa de IP reservada",
 "label.reservedsystemendip": "Fim dos IPs reservados para o sistema",
 "label.reservedsystemgateway": "Gateway de sistema reservado",
 "label.reservedsystemnetmask": "M\u00e1scara de rede reservada do sistema",
 "label.reservedsystemstartip": "In\u00edcio dos IPs reservados para o sistema",
+"label.reset": "Reiniciar",
 "label.reset.ssh.key.pair": "Recriar par de chaves SSH",
 "label.reset.ssh.key.pair.on.vm": "Recriar par de chaves SSH na VM",
-"label.reset.vpn.connection": "Resetar a conex\u00e3o VPN",
+"label.reset.to.default": "Reinicializar para o padr\u00e3o",
+"label.reset.vpn.connection": "Reiniciar a conex\u00e3o VPN",
 "label.resetvm": "Restabelecer VM",
 "label.resource": "Recurso",
-"label.resource.limit.exceeded": "Limite de Recurso Excedido",
-"label.resource.limits": "Limite de Recursos",
-"label.resource.name": "Nome do Recurso",
-"label.resourceid": "ID do Recurso",
-"label.resourcename": "Nome do Recurso",
+"label.resource.limit.exceeded": "Limite de recurso excedido",
+"label.resource.limits": "Limite de recursos",
+"label.resource.name": "Nome do recurso",
+"label.resourceid": "ID do recurso",
+"label.resourcename": "Nome do recurso",
 "label.resources": "Recursos",
-"label.resourcestate": "Estado do Recurso",
-"label.response.timeout.in.sec": "Timeout de Resposta (em seg)",
+"label.resourcestate": "Estado do recurso",
+"label.response.timeout.in.sec": "Timeout de resposta (em seg)",
 "label.restart.network": "Reiniciar rede",
-"label.restart.vpc": "reiniciar a VPC",
+"label.restart.vpc": "Reiniciar a VPC",
 "label.restartrequired": "Reiniciar obrigat\u00f3rio",
 "label.restore": "Restaurar",
+"label.restore.volume.attach": "Restaurar volume e anex\u00e1-lo",
 "label.retry.interval": "Intervalo de repeti\u00e7\u00e3o",
 "label.review": "Revisar",
 "label.revoke.project.invite": "Revogar convite",
+"label.revokeinvitationconfirm": "Por favor, confirme que voc\u00ea gostaria de revogar este convite?",
```

Comment:
It does not sound like a question, but more a notification like "Please, confirm that you would like to revoke".
I would remove the question mark.